airbyte-ci/connectors/metadata_service/lib/metadata_service/commands.py

```diff
@@ -14,12 +14,10 @@
 def log_metadata_upload_info(metadata_upload_info: MetadataUploadInfo):
     for file in metadata_upload_info.uploaded_files:
         if file.uploaded:
-            click.secho(
-                f"The {file.description} file for {metadata_upload_info.metadata_file_path} was uploaded to {file.blob_id}.", fg="green"
-            )
+            click.secho(f"File:{file.id} for {metadata_upload_info.metadata_file_path} was uploaded to {file.blob_id}.", fg="green")
```

Comment:
I think this has something to do with the fact that we're executing in Dagster, but why are we using `click.secho` here? /cc @erohmensing (I know the change here is just formatting, too, so question out of curiosity.)

Reply:
WDYM? Do you mean as compared to plain `echo`? Just for styling: https://click.palletsprojects.com/en/8.1.x/utils/#ansi-colors